fastlane/Fastfile

```diff
@@ -114,6 +119,72 @@ lane :release_purchases_ui_flutter do |options|
   end
 end
 
+desc "Create BC7 variant release"
+lane :release_bc7 do |options|
+  # Modify files in-place for BC7 variant
+  UI.message("Creating BC7 variant of purchases_flutter in-place...")
+
+  # Apply BC7 modifications
+  apply_bc7_modifications('..', false)
+
+  # Temporarily commit changes to satisfy git clean check
+  Dir.chdir('..') do
+    sh('git', 'add', 'pubspec.yaml', 'android/build.gradle')
+    sh('git', 'commit', '-m', '[TEMPORARY] BC7 modifications for publishing')
+  end
+
+  # Dry run
+  Dir.chdir('..') do
```

Comment:
I was just thinking it might be nice to be able to reuse existing code a bit more, for example calling the regular `release` lane here. But I also realized this would often require passing some new parameters and would probably not be worth the effort. Since it should be temporary in the grand scheme of things anyways I think this approach makes sense as-is.